### ISSUE TITLE
Shapeless Tuples exercises in website is pointing to Stdlib Tuples exercises

### ISSUE THREAD

**Issue by logistark** [scala-exercises/scala-exercises#458]
Shapeless Tuples exercises in the website show you, the stdlib tuples exercises. Also, Github links of this page are broken, since it redirects you to https://github.com/scala-exercises/exercises-shapeless/blob/master/src/main/scala/stdlib/Tuples.scala.

**Comment by dialelo** [comment]
Thanks for reporting, looking into it.

**Comment by dialelo** [comment]
I think I got the reason why this is happening. I've tried removing the stdlib exercises jar from the site and accessed to the Tuples section of the Shapeless exercises and the content is what I expected.

The problem is, I believe, in the generated source code with the library/section/exercise metadata. We are currently creating an `object` under the `defaultLib` package for every library, section and exercise. The problem is that we are not uniquifying the names of these objects, so both the stdlib's Tuples section and the shapeless Tuples section end up having the same name: `Section_tuple$1`. Since they both live in the same package (`defaultLib`) one overrides the other.

My approach is going to be to  generate the library/section/exercise metadata in a different package for every library, which could be something along the lines of `package scalaExercises.<library>` so they don't clash. This will require changes in the `compiler`, `sbt-exercise` and `runtime` project but shouldn't be a difficult fix.
